mentat/errors.py

```diff
@@ -3,20 +3,23 @@ class ModelError(Exception):
     pass
 
 
-# Used to indicate an issue with Mentat's code
-class MentatError(Exception):
+# Used to indicate a KeyboardInterupt thrown by a remote client
+class RemoteKeyboardInterrupt(Exception):
     pass
 
 
-# Used to indicate an issue with the user's usage of Mentat
-class UserError(Exception):
+# These 2 errors will have their message displayed in red, but no stacktrace thrown
+
+
+# TODO: Combine MentatError and UserError into just MentatError
+class MentatError(Exception):
     pass
 
 
-# Used to indicate a KeyboardInterupt thrown by a remote client
-class RemoteKeyboardInterrupt(Exception):
+class UserError(Exception):
     pass
 
 
+# This will exit the session without showing any sign of error
 class SessionExit(Exception):
     pass
```

Comment:
Instead of comments we can make these docstrings (will show up in VSCode/LSP intellisense descriptions).

E.g.

```
class RemoteKeyboardInterrupt(Exception):
   """Used to indicate a KeyboardInterrupt thrown by a remote client"""
```

Reply:
Good catch!